starter-code/main.js

```diff
@@ -0,0 +1,188 @@
+const MongoDB = require('mongodb');
+const mongoClient = MongoDB.MongoClient;
+const clear = require('clear');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const url = 'mongodb://localhost:27017/crounchbase';
+
+mongoClient.connect(url, (error, db) => {
+  if(error){
+    console.log('Error trying to connect');
+    console.log(error);
+  }
+  else {
+    console.log('Connection established with database');
+
+    function mainMenu(){
+      clear();
+      printMenu();
+      rl.question('Type an option: ', (option) =>{
+        switch(option){
+          case "1":
+            db.collection('companies').find({}, {name:1, _id: 0}).toArray((error, result) => {
+              if(error){
+                console.log('\ntype enter to continue ', (answer) =>{mainMenu()});
+                rl.question('\nType enter to continue: ', (answer) => {mainMenu()});
+              } else {
+                console.log(result);
+                rl.question('\nType enter to continue: ', (answer) => {mainMenu()});
+              }
+            });
+            break;
+          case "2":
+          db.collection('companies').find({}, {name:1, _id: 0}).count((error, result) => {
+            if(error){
+              console.log('\ntype enter to continue ', (answer) =>{mainMenu()});
+              rl.question('\nType enter to continue: ', (answer) => {mainMenu()});
+            } else {
+              console.log(result);
+              rl.question('\nType enter to continue: ', (answer) => {mainMenu()});
+            }
+          });
+            break;
+          case "3":
+            db.collection('companies').find({founded_year: 2004}, {name:1, _id: 0}).toArray((error, result) => {
```

Comment:
Here you want to count the number of elements that the query returns, use count (instead of find) again.